lib/ui/pages/location_page.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:food_app/utils/app_color.dart';
+import 'package:food_app/utils/app_typography.dart';
+
+class LocationPage extends StatelessWidget {
+  const LocationPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+        body: Padding(
+          padding: const EdgeInsets.symmetric(vertical: 20,horizontal: 22),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Image.asset("assets/images/binocular girl.png"),
+              Column(
+                children: [
+                  Text("Hi, nice to meet you!", style: AppTypography.blackw400.copyWith(fontSize: 22),textAlign: TextAlign.center,),
+                  Text("Set your location to start exploring restaurants around you", style: AppTypography.greyw400.copyWith(fontSize: 18),textAlign: TextAlign.center,)
+                ],
+              ),
+              Column(
+                children: [
+                  ElevatedButton(onPressed: (){
```

Comment:
We can use the `BaseButton` widget that has already been created in `widgets/app_button.dart` instead of using `ElevatedButton`. You can also give an enhancement to the BaseButton if it needed